pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/RebalanceTableCommand.java

```diff
@@ -18,32 +18,47 @@
  */
 package org.apache.pinot.tools.admin.command;
 
+import org.apache.pinot.common.utils.JsonUtils;
+import org.apache.pinot.controller.helix.core.rebalance.RebalanceResult;
 import org.apache.pinot.tools.Command;
-import org.apache.pinot.tools.PinotSegmentRebalancer;
+import org.apache.pinot.tools.PinotTableRebalancer;
 import org.kohsuke.args4j.Option;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 
+/**
+ * A sub-command for pinot-admin tool to rebalance a specific table
+ */
+@SuppressWarnings({"FieldCanBeLocal", "unused"})
 public class RebalanceTableCommand extends AbstractBaseAdminCommand implements Command {
```

Comment:
Make sure that the documentation is align with this command. I think that you deleted the command that @siddharthteotia recently added.

Reply:
looks like @Jackie-Jiang has moved RebalanceCommand and its functionality to RebalanceTableCommand. We just need to ensure that the functionality provided by the RebalanceCommand is also manually tested by running RebalanceTableCommand with pinot-admin tool

Reply:
The tests also covered increasing the number of replicas, decreasing the number of replicas, direct transition to target ideal state, incremental transition to ideal state and tests for realtime tables as well.

Reply:
I removed the old obsoleted one and renamed the one @siddharthteotia added to RebalanceTableCommand as the subcommand name is "RebalanceTable". Git cannot identify these changes, thus shown as this

Reply:
Sounds good.